contrib/apelscripts/condor_batch_blah.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+fail () {
+    echo "$@" >&2
+    exit 1
+}
+
+safe_config_val () {
+    var=$1
+    attr=$2
+    val=$(condor_ce_config_val $attr) ||
+    fail "Failed to retrieve CE configuration value '$attr'"
+    eval "$var"='$val'
+}
+
+OUTPUT_DIR="$(condor_ce_config_val APEL_OUTPUT_DIR)"
+[[ -d $OUTPUT_DIR && -w $OUTPUT_DIR ]] || fail "Cannot write to $OUTPUT_DIR"
+OUTPUT_DATETIME=`date '+%Y%m%d-%H%M'`
+
+HISTORY_DIR=`condor_config_val PER_JOB_HISTORY_DIR`
+QUARANTINE_DIR=$HISTORY_DIR/quarantine
+[[ -d $QUARANTINE_DIR && -w $QUARANTINE_DIR ]] || fail "Cannot write to $QUARANTINE_DIR"
+
+HISTORY_EXTRA_ARGS=(-format "\n" EMPTY)
+safe_config_val SCALING_ATTR APEL_SCALING_ATTR
+[[ -z $SCALING_ATTR ]] || HISTORY_EXTRA_ARGS=(-format "%v|" "${SCALING_ATTR}" "${HISTORY_EXTRA_ARGS[@]}")
+
+safe_config_val BATCH_HOST APEL_BATCH_HOST
+safe_config_val CE_HOST APEL_CE_HOST
+safe_config_val CE_ID APEL_CE_ID
+
+# Iterate over all the files in PER_JOB_HISTORY_DIR
+for file in $HISTORY_DIR/*
+do
+    # If $file is a directory, skip it
+    if [[ -d $file ]]; then
+        continue
+    fi
+
+    # Check if $file is a valid history file by looking for a ClusterId value
+    # If not found, assume the file is invalid and move to quarantine folder
+    if [[ -z "`condor_q -job $file -format \"%s\" ClusterId`" ]]; then
```

Comment:
```suggestion
    if [[ -z `condor_q -job $file -format "%s" ClusterId` ]]; then
```

You shouldn't need the `"` inside double brackets